src/vmm/src/logger/metrics.rs

```diff
@@ -1239,7 +1239,17 @@ mod tests {
 
     #[test]
     fn test_init() {
-        let m = &METRICS;
+        // let m = &METRICS;
+        // Now that we have moved logger to vmm, this test
+        // has a conflict with the vmm_config test `test_init_metrics`
+        // which also uses "METRICS" and tests fail with an already
+        // initialized error.
+        // This test is to validate the init() which doesn't require
+        // using METRICS specifically. So, to avoid the conflict we
+        // use a local Metrics to test init() instead of the global
+        // "METRICS"
```

Comment:
First line is left-over commented code. I'd also rephrase the contents to only refer to the current state of the code (e.g. remove the "now that we moved logger to vmm").